src/convertToObject.js

```diff
@@ -3,16 +3,32 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const arr = sourceString.split(';');
+
+  const obj = arr.reduce((accumulator, element) => {
+    const separator = element.indexOf(':');
+    const key = element.slice(0, separator).trim();
+    const value = element.slice(separator + 1).trim();
+
+    accumulator[key] = value;
+
+    return accumulator;
+  }, {});
+
+  delete obj[''];
```

Comment:
why do we need it? you can check your key before adding it to accumulator

Reply:
done